xbmc/cores/VideoPlayer/VideoRenderers/WinRenderer.cpp

```diff
@@ -911,38 +979,28 @@ bool CWinRenderer::RenderCapture(CRenderCapture* capture)
 //********************************************************************************************************
 // YV12 Texture creation, deletion, copying + clearing
 //********************************************************************************************************
-void CWinRenderer::DeleteYV12Texture(int index)
+void CWinRenderer::DeleteRenderBuffer(int index)
 {
   CSingleLock lock(g_graphicsContext);
   ReleaseBuffer(index);
-  SAFE_DELETE(m_VideoBuffers[index]);
+  m_renderBuffers[index].Release();
```

Comment:
it's not the same. `ReleaseBuffer` releases video buffer only. `Release` method releases render buffer resources. perhaps need to change name for better understanding.